### PR TITLE
AVX-456: Add bayer format renderer

### PR DESCRIPTION
**Pull request by dat-tran-matternet**

### CODE REVIEW

File: src/host/libpixyusb_swig/get_exposure.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+##
+# @file get_exposure.py
+# @brief This tool reads the Pixy's gain/exposure settings after certain program settings of the Pixy.
+#        The tool revealed a gain/exposure inconsistency in IR-Lock's MarkOne firmware. When used in-flight the
+#        MarkOne's gain/exposure is 0xFE and 0x01 respectively. When used with the computer application Pixymon, the
+#        gain/exposure changes to 0x00 and 0x20 respectively. The most likely reason for this inconsistency is because
+#        with the in-flight setting of 0x01FE, the displayed image on Pixymon has red/blue artifacts due to shifting of
+#        the bayer pattern. It is not clear why the camera sensor has this shift when the gain is high and exposure is
+#        low.
```
Comment (rabad-matternet) [inline src/host/libpixyusb_swig/get_exposure.py:11]: Please forgive my ignorance.... what is the context in which this python script will execute?  We run it once after the Pixy is configured?  Or is it more often than that?

Comment (dat-tran-matternet) [reply src/host/libpixyusb_swig/get_exposure.py:11]: I developed this script to prove that the IR Lock's MarkOne firmware is using two different gain/exposure settings. One setting when used with Pixymon (like a USB webcam) and one setting when used via I2C (in flight) with the Cube. 

No one should need to run this but if they want proof then they can run it. If not they can take my word for it. =)

File: src/host/libpixyusb_swig/pixy_viewer.py [hunk 2]
```diff
@@ -36,48 +36,136 @@
...
+NEIGHBORS_DIRECTION = [
+    (0, 0),  # center
+    (-1, 0),  # top
+    (-1, 1),  # top-right
+    (0, 1),  # right
+    (1, 1),  # bottom-right
+    (1, 0),  # bottom
+    (1, -1),  # bottom-left
+    (0, -1),  # left
+    (-1, -1),  # top-left
```
Comment (rabad-matternet) [inline src/host/libpixyusb_swig/pixy_viewer.py]: Is this an (X,Y) coordinate system?

Comment (dat-tran-matternet) [reply src/host/libpixyusb_swig/pixy_viewer.py]: It's  row then column of an image which would translate to (y, x).  An image is scanned a row at a time from left to right and top to bottom

Comment (rabad-matternet) [reply src/host/libpixyusb_swig/pixy_viewer.py]: Yeah, I thought it was (Y, X) as well.
Can you put a comment stating that?

**Review by rabad-matternet — APPROVED** [review]
Please see my comment about (Y, X).

**Review by zalice — APPROVED** [review]
🔥

**Pull request merged**